quiz.js

```diff
@@ -37,7 +51,27 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'stark'
+}, {
+  $push: { people: {
+    peopleName: 'ned', 'arya', 'sansa'}
+  }
+});
+
+db.houses.update({
+  name: 'Targaryen'
+}, {
+  $push: { peopleName: 'Viserys', 'Daenerys'}
+  }
+});
 
+db.houses.update({
+  name: 'Arryn'
+},{
+  $push: {peopleName: 'Jon'}
+  }
+});
```

Comment:
This one is correct - for a single value, the syntax is 
```
db.houses.update({ name : 'arryn' }, { $push : { peopleName : 'Jon' } });
```
(As an aside, though, `peopleNames` would probably be a better name for that property, since it might contain multiple names.)